src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+            for (int i = 0; i < 3; i++) {
```

Comment:
The number `3` used in the loop is a magic number. According to the checklist, all magic numbers should be constants. Consider defining a constant for this value.